packages/edit-post/src/style.scss

```diff
@@ -1,6 +1,6 @@
 $footer-height: $button-size-small;
 
-@import "./components/fullscreen-mode/style.scss";
+@import "../../block-editor/src/components/fullscreen-mode/style.scss";
```

Comment:
@youknowriad I'm thinking these styles should probably be moved out of `block-editor` and every editor implementation should be able to specify their own styles when `is-fullscreen-mode` class is added by `FullscreenMode` component. Since both `edit-post` and `edit-site` are running in the same wp-admin context, we'd probably end up duplicating the code from it, but that might be fine. What do you think?

Reply:
I agree with the moving, for the global style, maybe it's fine to just pick one as a convention.

Reply:
Based in our Slack discussion I moved this import to `block-editor` package in 9e2b8deb5321b3e7353367421721c90d03eeab53.